dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyConnectionHandler.java

```diff
@@ -61,10 +61,6 @@ public void onGoAway(Object channel) {
         if (connectionClient != null) {
             connectionClient.onGoaway(nettyChannel);
         }
-        if (LOGGER.isDebugEnabled()) {
-            LOGGER.debug(String.format("Channel %s go away ,schedule reconnect", nettyChannel));
-        }
-        reconnect(nettyChannel);
```

Comment:
One question is if go away does not reconnect here, will it reconnect after the tcp connection is disconnected? Because in some cases the registration center will not push information about going offline and going online again.

Reply:
GoAway Frame is used to handle server connection closure graceful, if client don't know the change of service list, it should be complete with heartbeat task, refer: #12955